assets/components/progressMessage/progressMessage.jsx

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+
+// ---- Types ----- //
+
+type PropTypes = {
+  message: string,
+};
+
+
+// ----- Component ----- //
+
+
+export default function ProgressMessage(props: PropTypes) {
+  return (
+    <div className="component-progress-message">
+      <span>{props.message}</span>
+    </div>
+  );
+}
```

Comment:
This component is temporary, until Frank designs a spinner.  It will be easier to throw this small component away when we decide what we are actually doing.